docs/user_guide/assessment/common_fairness_metrics.rst

```diff
@@ -299,7 +303,122 @@ capture the costs of missclassification disparities.
 
 
 
+.. _assessment_four_fifths:
+
+The Four Fifths Rule: Often Misapplied
```

Comment:
Following the discussion of the paper from the folks at Spotify, I can imagine that this would at some point become a subsection of a new section "Determine thresholds"/"How fair is fair enough"/whatever we want to call it under "Performing a fairness assessment". But for now I'm also happy to have this here.

Reply:
Yeah, with the current state of the documentation I think the place in your PR makes sense.